readme.md

```diff
@@ -45,6 +45,12 @@ await conn.close();
 For the full documentation see the
 [edgedb-js docs](https://www.edgedb.com/docs/clients/01_js/index).
 
+## TLS
+
+deno doesn't currently support TLS ALPN for client connections. This means that
+to use deno with the latest version of edgedb the server needs to be run
+without TLS which is on by default.
```

Comment:
We need to explain how, which is by supplying the server with `--allow-insecure-binary-clients` command line arg, or setting the `EDGEDB_SERVER_ALLOW_INSECURE_BINARY_CLIENTS` env var

Reply:
We also must say that deno support is experimental for this very reason.

Reply:
updated